lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +242,57 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        conllu_sentences = []
+        mystem = Mystem()
+        sentences = split_by_sentence(text)
+        for sent_idx, sentence in enumerate(sentences):
+            print(sentence)
+            conllu_tokens = []
+            tokens = [token for token in mystem.analyze(sentence) if
+                      token['text'].strip() and token['text'] not in string.punctuation.replace('.', '')]
+            for token_idx, token_info in enumerate(tokens):
+                token_text = token_info['text']
+                print(token_text)
+                token = ConlluToken(token_text)
+                token.set_position(token_idx)
+
+                if token_info.get('analysis'):
+                    lemma = token_info['analysis'][0]['lex']
+                    pos = re.match('^[^,|=]*', token_info['analysis'][0]['gr']).group(0)
```

Comment:
for pos processing you need to use mystem converter

Reply:
done